deployment/environments/production/services/api-ingress.yml

```diff
@@ -0,0 +1,65 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: api-ingress
+  annotations:
+    kubernetes.io/ingress.global-static-ip-name: api-ingress
+spec:
+  defaultBackend:
+    service:
+      name: api
+      port:
+        number: 80
+  # rules:
```

Comment:
One thing to keep in mind here is that the CLI currently I believe references these rules to create features branches (ie `feature-api.foo.com` etc)... I would like this to still work if possible.

Reply:
Also if not needed then this can just be removed?

Reply:
I believe the CLI requires an Ingress if you want to create feature branch and update the rules. I haven't tested this. The current default ( before this PR) is no ingress at all.

Reply:
I'm also looking into [Gateways](https://cloud.google.com/kubernetes-engine/docs/concepts/gateway-api) which seem to be the successor for Ingresses.

Reply:
oh that's interesting... Well I have some things I want to do with the CLI as well I guess I can make sure this is still working once this is merged and maybe sync with you on gateways 👍